src/Drivers/SMS77.php

```diff
@@ -0,0 +1,167 @@
+<?php
+namespace SimpleSoftwareIO\SMS\Drivers;
+
+use GuzzleHttp\Client;
+use SimpleSoftwareIO\SMS\DoesNotReceive;
+use SimpleSoftwareIO\SMS\MakesRequests;
+use SimpleSoftwareIO\SMS\OutgoingMessage;
+
+class SMS77 extends AbstractSMS implements DriverInterface
+{
+    use DoesNotReceive, MakesRequests;
+
+    /**
+     * The API's URL.
+     *
+     * @var string
+     */
+    protected $apiBase = 'http://gateway.sms77.de/';
+
+    /**
+     * The Guzzle HTTP Client.
+     *
+     * @var \GuzzleHttp\Client
+     */
+    protected $client;
+
+    /**
+     * Create the SMS77 instance.
+     *
+     * @param Client $client The Guzzle Client
+     */
+    public function __construct(Client $client, $username, $password, $debug = false)
+    {
+        $this->client = $client;
+        $this->setUser($username);
+        $this->setPassword($password);
+        $this->setDebug($debug);
+    }
+
+    /**
+     * Sends a SMS message.
+     *
+     * @param \SimpleSoftwareIO\SMS\OutgoingMessage $message
+     */
+    public function send(OutgoingMessage $message)
+    {
+        $composeMessage = $message->composeMessage();
+
+        //Convert to sms77 format.
+        $numbers = implode(',', $message->getTo());
+
+        $data = [
+            'u' => $this->auth['username'],
+            'p' => $this->auth['password'],
+            'to' => $numbers,
+            'type' => 'direct',
+            'text' => $composeMessage,
+            'debug' => (int) $this->debug,
+        ];
+
+        $this->buildCall('');
+        $this->buildBody($data);
+
+        $response = $this->postRequest();
+        if ($this->hasError($response->getBody())) {
+            $this->handleError($response->getBody());
+        }
+        return $response;
+    }
+
+    /**
+     * Checks if the transaction has an error
+     *
+     * @param $body
+     * @return bool
+     */
+    protected function hasError($body)
+    {
+        if ($body != '100') {
+            return $body;
+        }
+        return false;
+    }
+
+    /**
+     * Log the error message which ocurred
+     *
+     * @param $body
+     */
+    protected function handleError($body)
+    {
+        $error = 'An error occurred. Status code: ' . $body . ' - ';
+
+        //From https://www.sms77.de/api.pdf Rückgabewerte (German doc)
+        switch($body){
+            case '101':
+                $error .= 'Versand an mindestens einen Empfänger fehlgeschlagen';
```

Comment:
It's the original messages from their API doc which seems to be only available in German... but a translation might do